tests/vppinputdecode.cpp

```diff
@@ -51,6 +51,14 @@ bool VppInputDecode::config(NativeDisplay& nativeDisplay)
     return status == DECODE_SUCCESS;
 }
 
+void getTimeStamp(SharedPtr<VideoFrame>& frame, VideoDecodeBuffer inputBuffer){
```

Comment:
I look at this again. simply remove first argument is ok, but you still need make this as member function.